apps/homescreen/js/pages.js

```diff
@@ -260,9 +488,135 @@
         }
         break;
 
+      // Disable scrolling during dragging, and display bottom-bar
+      case 'drag-start':
+        console.debug('Drag-start on ' + e.detail.target.dataset.id);
```

Comment:
Let's remove these console.debug calls - the ones in apps.js were added to try to debug a bug that has since been marked as WORKSFORME.